src/graph_gui.cpp

```diff
@@ -567,6 +584,18 @@ struct BaseGraphWindow : Window {
 		SetWindowDirty(WC_GRAPH_LEGEND, 0);
 	}
 
+	void Blink()
+	{
+		if (this->highlight_data == UINT8_MAX && this->highlight_range == UINT8_MAX) return;
+
+		this->highlight_state = !this->highlight_state;
+		this->SetDirty();
+	}
+
+	const IntervalTimer<TimerWindow> blink_interval = {std::chrono::milliseconds(450), [this](auto) {
```

Comment:
We have other things that blink, presumably at the same rate. Could it use a constant or do the others use different units? (Maybe it's worth making a new constant anyway, perhaps in a timer header where it's easy to access.)